coriolis/schemas/vm_export_info_schema.json

```diff
@@ -110,8 +110,7 @@
             },
             "required": [
               "format",
```

Comment:
Could you please also remove 'format' from the list of required params? It won't be needed either.